File: ProgrammingFundamentals/Java/02.ConditionalStatementsAndLoops/Exercises/src/P14_MagicLetter.java
import java.util.Scanner;
public class P14_MagicLetter {
    public static void main(String[] args) {
        Scanner console = new Scanner(System.in);
        char a = console.nextLine().charAt(0);
        char b = console.nextLine().charAt(0);
        char c = console.nextLine().charAt(0);

        for(char i = a; i <= b; i++) {
            for(char j = a; j <= b; j++) {
                for(char k = a; k <= b; k++) {
                    boolean remLetter3 = (i != c) && (j != c) && (k != c);
                    if(remLetter3) {
                        System.out.print("" + i + j + k + " ");
                    }
                }
            }
        }
    }
}

File: ProgrammingFundamentals/C#/02.ConditionalStatementsAndLoops/Exercises/P14_MagicLetter/P14_MagicLetter.cs
using System;

namespace P14_MagicLetter
{
    class P14_MagicLetter
    {
        static void Main(string[] args)
        {
            char a = char.Parse(Console.ReadLine());
            char b = char.Parse(Console.ReadLine());
            char c = char.Parse(Console.ReadLine());

            for (char i = a; i <= b; i++)
            {
                for (char j = a; j <= b; j++)
                {
                    for (char k = a; k <= b; k++)
                    {
                        bool remLetter3 = (i != c) && (j != c) && (k != c);
                        if (remLetter3)
                        {
                            Console.Write("" + i + j + k + " ");
                        }
                    }
                }
            }
        }
    }
}
